scout/parse/case.py

```diff
@@ -41,6 +41,41 @@ def get_correct_date(date_info):
     return datetime.datetime.now()
 
 
+def parse_custom_images(config_data):
+    """Parse information on custom images assigned to the case."""
+    VALID_IMAGE_SUFFIXES = ["svg", "png", "jpeg"]
+    LOG.debug("Parse custom images")
+
+    # sort custom image sections
+    parsed_sections = {}
+    for section_name, images in config_data.get("custom_images", {}).items():
+        parsed_images = []
+        for image in images:
+            # skip entries that are not recognized as image on suffix
+            path = Path(image["path"])
+            if path.suffix in VALID_IMAGE_SUFFIXES:
```

Comment:
Here are two bugs.

1. .suffix() returns ".png" not "png"
2. if suffix is valid, the current loop would exit.